eng/pipelines/apiview-review-gen-swagger.yml

```diff
@@ -27,7 +27,7 @@ jobs:
   - script: >
       dotnet tool install
       Azure.Sdk.Tools.SwaggerApiParser
-      --version 1.0.7-dev.20230211.1
+      --version 1.0.7-dev.20230209.5
```

Comment:
This seems more like a downgrade than an upgrade.

Reply:
I copied the one from the pipeline. Looking

Reply:
Updated in the pipeline and in this PR